tests/functional/spec/census/issues/routing-last-week-were-you.spec.js

```diff
@@ -0,0 +1,158 @@
+import NumberOfBedrooms from '../../../pages/surveys/census/household/number-of-bedrooms.page.js'
+import CentralHeating from '../../../pages/surveys/census/household/central-heating.page.js'
+import OwnOrRent from '../../../pages/surveys/census/household/own-or-rent.page.js'
+import NumberOfVehicles from '../../../pages/surveys/census/household/number-of-vehicles.page.js'
+import HouseholdAndAccommodationCompleted from '../../../pages/surveys/census/household/household-and-accommodation-completed.page.js'
+import HouseholdMemberBegin from '../../../pages/surveys/census/household/household-member-begin.page.js'
+import DetailsCorrect from '../../../pages/surveys/census/household/details-correct.page.js'
+import Sex from '../../../pages/surveys/census/household/sex.page.js'
+import DateOfBirth from '../../../pages/surveys/census/household/date-of-birth.page.js'
+import Over16 from '../../../pages/surveys/census/household/over-16.page.js'
+import MaritalStatus from '../../../pages/surveys/census/household/marital-status.page.js'
+import AnotherAddress from '../../../pages/surveys/census/household/another-address.page.js'
+import InEducation from '../../../pages/surveys/census/household/in-education.page.js'
+import CountryOfBirth from '../../../pages/surveys/census/household/country-of-birth.page.js'
+import Carer from '../../../pages/surveys/census/household/carer.page.js'
+import NationalIdentity from '../../../pages/surveys/census/household/national-identity.page.js'
+import EthnicGroup from '../../../pages/surveys/census/household/ethnic-group.page.js'
+import WhiteEthnicGroup from '../../../pages/surveys/census/household/white-ethnic-group.page.js'
+import UnderstandWelsh from '../../../pages/surveys/census/household/understand-welsh.page.js'
+import Language from '../../../pages/surveys/census/household/language.page.js'
+import Religion from '../../../pages/surveys/census/household/religion.page.js'
+import PastUsualAddress from '../../../pages/surveys/census/household/past-usual-address.page.js'
+import Passports from '../../../pages/surveys/census/household/passports.page.js'
+import Disability from '../../../pages/surveys/census/household/disability.page.js'
+import Qualifications from '../../../pages/surveys/census/household/qualifications.page.js'
+import Volunteering from '../../../pages/surveys/census/household/volunteering.page.js'
+import EmploymentType from '../../../pages/surveys/census/household/employment-type.page.js'
+import Jobseeker from '../../../pages/surveys/census/household/jobseeker.page.js'
+
+describe('ArriveInUk', function () {
+  it('Given I am answering question 28.	Last week were you.., When I dont not select any response, Then I am routed to 29. Were you actively looking for...?', function () {
+      openAndStartCensusQuestionnaire('census_household.json')
+
+      // who-lives-here
+      PermanentOrFamilyHome
```

Comment:
I agree, But given these issue tests are going to be short lived I'm not going to loose a lot of sleep over it.
It will be one to raise in the next testing review?